src/test/java/hw1/TestingDiv.java

```diff
@@ -0,0 +1,69 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestingDiv {
```

Comment:
I could see that in the following classes you have code duplication:
- TestingDiv
- TestingMul
- TestingSub
- TestingSum
May be it will be better extract them?